src/lib/ContractParser.js

```diff
@@ -160,14 +161,31 @@ export class ContractParser {
     }
     let interfaces
     if (isErc165) interfaces = await this.getInterfacesERC165(contract)
+    else if (this.isEIP1167(txInputData)) interfaces = { EIP1167: true };
     else interfaces = this.getInterfacesByMethods(methods)
     interfaces = Object.keys(interfaces)
       .filter(k => interfaces[k] === true)
       .map(t => contractsInterfaces[t] || t)
     return { methods, interfaces }
   }
 
-  async getInterfacesERC165 (contract) {
+  getMasterCopy(bytecode) {
+    let newByteCode = bytecode.replace('363d3d373d3d3d363d73', '');
+    newByteCode = newByteCode.replace('5af43d82803e903d91602b57fd5bf3', '');
+    console.log('newByteCode:',newByteCode);
+    const re = new RegExp('^0x[a-fA-F0-9]{40}$', 'i');
+    if (re.test(newByteCode))
```

Comment:
Why this would return `false`? I guess we want to use this function only if the contract is EIP-1167